### ISSUE TITLE
Divider doesn't match screenshot in readme

### ISSUE THREAD

**Issue by ezzak** [JureSotosek/ink-divider#4]
I'm not sure the divider is really a divider anymore...

![image](https://user-images.githubusercontent.com/20268283/61575049-f46d1680-aa7b-11e9-8969-a9a768fe2daf.png)

**Comment by JureSotosek** [comment]
Huh? I just tried the examples and this is how they show for me:
<img width="397" alt="Screenshot 2019-07-20 at 13 43 07" src="https://user-images.githubusercontent.com/16746406/61578261-5b2e1680-aaf4-11e9-8471-d1f8a7a07758.png">

Can you post the `props` / the code you used to achieve this?

**Comment by ezzak** [comment]
```
<React.Fragment>
  <Divider title="Test" width={20} />
  {...otherComponents}
</React.Fragment>
```

^ Above rendered via `render(<MyComponent />, process.stderr)`

VSCode terminal:

![image](https://user-images.githubusercontent.com/20268283/61582535-f2896e80-aae0-11e9-980d-93116d729144.png)

iTerm2:
![image](https://user-images.githubusercontent.com/20268283/61582544-06cd6b80-aae1-11e9-9ba6-9eb6fdaccdea.png)

**Comment by JureSotosek** [comment]
Rendering this:
``` javascript
import React from 'react';
import { render } from 'ink';
import Divider from '..';

const Basic = () => (
  <React.Fragment>
    <Divider title="Test" width={20} />
  </React.Fragment>
);

render(<Basic />);
```

Gives me this:

<img width="652" alt="Screenshot 2019-07-21 at 21 27 11" src="https://user-images.githubusercontent.com/16746406/61596019-63727880-abfe-11e9-8b32-3a0c9b16e257.png">

Can you link me a project or something where this happens? That would be really helpful ✌🏼

**Comment by ezzak** [comment]
https://repl.it/@ezzak/ink-divider

![image](https://user-images.githubusercontent.com/20268283/61596246-d5729f80-abb5-11e9-82ad-ea2f80e9344f.png)

**Comment by JureSotosek** [comment]
Hmmm, the problem seems to be when switching from `ink@2.0.0` to `ink@2.1.0`. Not sure why, but in that version, `Color` element switched from being an inline element to being a block element. Not sure if this is the intended behavior or a bug.

Will push a quick update that will fix this issue.

**Comment by JureSotosek** [comment]
Bug fixed with `ink-divider@2.0.1` 👍🏼
